bin/pint_db.py

```diff
@@ -0,0 +1,402 @@
+# Copyright (c) 2021 SUSE LLC
```

Comment:
The following is not something I'm suggesting needs to be changed, just some feedback on the toolchain/workflow...


This script doesn't have a `#!` line, so it isn't really an "executable" so technically shouldn't be located in a `bin` directory... The original orm_exporttopostgres was an executable script because it was located in a `bin` directory...

Also everything that you are doing below in the wrapper shell script can be done in this python script itself, e.g. checking for the presence of the VIRTUAL_ENV env var, and dynamically updating the sys.path list to include the public-cloud-info-service...

And the latter would not be required if the create_dev_venv.sh script was modified to do a `pip install $SRCDIR` to ensure that the `pint_server` module is also installed into the virtualenv... Or better yet `pip install -e $SRCDIR`, i.e. developer mode install using a symlink to the $SRCDIR rathern than copying the contends, so that you don't have to keep re-installing if you are making development changes ;-)

In the long run it might be good to just make this script an executable script, which works with whatever the active runtime environment is. You could have the default value for the --repository option be the path that is dynamically calculated relative to the path of this script, similar to the determination of the `pint-data` repo location. ;-)

Reply:
that's a good point. Let me see what I can do.